front/lib/api/poke/plugins/data_sources/operations.ts

```diff
@@ -0,0 +1,73 @@
+import { assertNever, ConnectorsAPI, Err, Ok } from "@dust-tt/types";
+
+import config from "@app/lib/api/config";
+import { createPlugin } from "@app/lib/api/poke/types";
+import { DataSourceResource } from "@app/lib/resources/data_source_resource";
+import logger from "@app/logger/logger";
+
+const OPERATIONS = ["STOP", "PAUSE", "UNPAUSE", "RESUME", "SYNC"] as const;
+
+const doOperation = (op: (typeof OPERATIONS)[number], connectorId: string) => {
```

Comment:
IIRC, our CLI is exposed through the connector API as well, don't we want to use it?

Reply:
The /admin endpoint would be interesting to add specific plugins per provider type, but commands have to be whitelisted to be callable via api , and some commands require more parameters - let's keep that for later